Test Failure: io.openliberty.http.monitor.fat.ContainerJSFApplicationTest.cjsf_testXHTMLext_EE11_FEATURES_MicroProfile_70_App_Mode

RTC 308673

```
cjsf_testXHTMLext_EE11_FEATURES_MicroProfile_70_App_Mode:java.lang.Throwable: 2026-02-18-05:26:22:874 500 Internal Server Error for http://localhost:8010/jsfApp/index.xhtml
    at componenttest.custom.junit.runner.FATRunner.newThrowableWithTimeStamp(FATRunner.java:386)
    at componenttest.custom.junit.runner.FATRunner.access$700(FATRunner.java:74)
    at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:343)
    at componenttest.rules.repeater.RepeatTests$CompositeRepeatTestActionStatement.evaluate(RepeatTests.java:149)
    at org.testcontainers.containers.FailureDetectingExternalResource$1.evaluate(FailureDetectingExternalResource.java:29)
    at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:412)
    at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:204)
Caused by: com.gargoylesoftware.htmlunit.FailingHttpStatusCodeException: 500 Internal Server Error for http://localhost:8010/jsfApp/index.xhtml
    at com.gargoylesoftware.htmlunit.WebClient.throwFailingHttpStatusCodeExceptionIfNecessary(WebClient.java:704)
    at com.gargoylesoftware.htmlunit.WebClient.getPage(WebClient.java:461)
    at com.gargoylesoftware.htmlunit.WebClient.getPage(WebClient.java:368)
    at com.gargoylesoftware.htmlunit.WebClient.getPage(WebClient.java:520)
    at com.gargoylesoftware.htmlunit.WebClient.getPage(WebClient.java:501)
    at io.openliberty.http.monitor.fat.ContainerJSFApplicationTest.cjsf_testXHTMLext(ContainerJSFApplicationTest.java:160)
    at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
    at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:242)
```

--- a/dev/io.openliberty.http.monitor_fat/publish/servers/ContainerJSFServer/server.xml
+++ b/dev/io.openliberty.http.monitor_fat/publish/servers/ContainerJSFServer/server.xml
@@ -5,6 +5,8 @@
     	<feature>mpTelemetry-2.0</feature>
     </featureManager>
 
+    <httpSession useSeparateSessionInvalidatorThreadPool="false" delayInvalidationAlarmDuringServerStartup="90"/>
+
     <include location="../fatTestPorts.xml" />
 
     <quickStartSecurity userName="theUser" userPassword="thePassword"/>

--- a/dev/io.openliberty.http.monitor_fat/publish/servers/ContainerJSPServer/server.xml
+++ b/dev/io.openliberty.http.monitor_fat/publish/servers/ContainerJSPServer/server.xml
@@ -11,6 +11,8 @@
 
     <include location="../fatTestPorts.xml" />
 
+    <httpSession useSeparateSessionInvalidatorThreadPool="false" delayInvalidationAlarmDuringServerStartup="90"/>
+
     <quickStartSecurity userName="theUser" userPassword="thePassword"/>
     <keyStore id="defaultKeyStore" password="Liberty"/>
 

--- a/dev/io.openliberty.http.monitor_fat/publish/servers/ContainerJustServer/server.xml
+++ b/dev/io.openliberty.http.monitor_fat/publish/servers/ContainerJustServer/server.xml
@@ -7,6 +7,8 @@
 
     <include location="../fatTestPorts.xml" />
 
+    <httpSession useSeparateSessionInvalidatorThreadPool="false" delayInvalidationAlarmDuringServerStartup="90"/>
+
     <quickStartSecurity userName="theUser" userPassword="thePassword"/>
     <keyStore id="defaultKeyStore" password="Liberty"/>
 

--- a/dev/io.openliberty.http.monitor_fat/publish/servers/ContainerRestServer/server.xml
+++ b/dev/io.openliberty.http.monitor_fat/publish/servers/ContainerRestServer/server.xml
@@ -5,6 +5,8 @@
     	<feature>mpTelemetry-2.0</feature>
     </featureManager>
 
+    <httpSession useSeparateSessionInvalidatorThreadPool="false" delayInvalidationAlarmDuringServerStartup="90"/>
+
     <include location="../fatTestPorts.xml" />
 
     <quickStartSecurity userName="theUser" userPassword="thePassword"/>

--- a/dev/io.openliberty.http.monitor_fat/publish/servers/ContainerServletServer/server.xml
+++ b/dev/io.openliberty.http.monitor_fat/publish/servers/ContainerServletServer/server.xml
@@ -5,6 +5,8 @@
     	<feature>mpTelemetry-2.0</feature>
     </featureManager>
 
+    <httpSession useSeparateSessionInvalidatorThreadPool="false" delayInvalidationAlarmDuringServerStartup="90"/>
+
     <include location="../fatTestPorts.xml" />
 
     <quickStartSecurity userName="theUser" userPassword="thePassword"/>

--- a/dev/io.openliberty.http.monitor_fat/publish/servers/JSFServer/server.xml
+++ b/dev/io.openliberty.http.monitor_fat/publish/servers/JSFServer/server.xml
@@ -5,6 +5,8 @@
     	<feature>mpMetrics-5.1</feature>
     </featureManager>
 
+    <httpSession useSeparateSessionInvalidatorThreadPool="false" delayInvalidationAlarmDuringServerStartup="90"/>
+
     <include location="../fatTestPorts.xml" />
 
     <quickStartSecurity userName="theUser" userPassword="thePassword"/>

--- a/dev/io.openliberty.http.monitor_fat/publish/servers/JSPServer/server.xml
+++ b/dev/io.openliberty.http.monitor_fat/publish/servers/JSPServer/server.xml
@@ -5,6 +5,8 @@
     	<feature>mpMetrics-5.1</feature>
     </featureManager>
 
+    <httpSession useSeparateSessionInvalidatorThreadPool="false" delayInvalidationAlarmDuringServerStartup="90"/>
+
     <include location="../fatTestPorts.xml" />
 
     <quickStartSecurity userName="theUser" userPassword="thePassword"/>

--- a/dev/io.openliberty.http.monitor_fat/publish/servers/MBeanServer/server.xml
+++ b/dev/io.openliberty.http.monitor_fat/publish/servers/MBeanServer/server.xml
@@ -8,6 +8,8 @@
     	<feature>monitor-1.0</feature>
     </featureManager>
 
+    <httpSession useSeparateSessionInvalidatorThreadPool="false" delayInvalidationAlarmDuringServerStartup="90"/>
+
     <include location="../fatTestPorts.xml" />
 
     <quickStartSecurity userName="theUser" userPassword="thePassword"/>

--- a/dev/io.openliberty.http.monitor_fat/publish/servers/SimpleRestServer/server.xml
+++ b/dev/io.openliberty.http.monitor_fat/publish/servers/SimpleRestServer/server.xml
@@ -5,6 +5,8 @@
     	<feature>mpMetrics-5.1</feature>
     </featureManager>
 
+    <httpSession useSeparateSessionInvalidatorThreadPool="false" delayInvalidationAlarmDuringServerStartup="90"/>
+
     <include location="../fatTestPorts.xml" />
 
     <quickStartSecurity userName="theUser" userPassword="thePassword"/>

--- a/dev/io.openliberty.http.monitor_fat/publish/servers/SimpleServletServer/server.xml
+++ b/dev/io.openliberty.http.monitor_fat/publish/servers/SimpleServletServer/server.xml
@@ -5,6 +5,8 @@
     	<feature>mpMetrics-5.1</feature>
     </featureManager>
 
+    <httpSession useSeparateSessionInvalidatorThreadPool="false" delayInvalidationAlarmDuringServerStartup="90"/>
+
     <include location="../fatTestPorts.xml" />
 
     <quickStartSecurity userName="theUser" userPassword="thePassword"/>

